src/randoop/main/GenInputsAbstract.java

```diff
@@ -366,36 +318,42 @@ public GenInputsAbstract(String command, String pitch,
    */
   @Option("Which exceptional tests to include in the regression test suite")
   public static ExceptionAssertionsMode regression_assertions_about_exceptions = ExceptionAssertionsMode.ALL;
-
-
-  // TODO make an enum. (But presently Options package requires upper-case
-  // strings for enums, which will require changes to Make targets, plugin, etc.)
-  /** For details, see the Javadoc documentation for {@link randoop.DefaultTestFilter}. */
-  @Option("What kinds of tests to output: pass, fail, or all")
-  public static String output_tests = "all";
   
-  public final static String ALL = "all";
-  public final static String FAIL = "fail";
-  public final static String PASS = "pass";
-
-  @Option("Simplify (shorten) failed tests while preserving failure behavior")
-  public static boolean simplify_failed_tests = false;
-
+  /**
+   * The possible values for exception allocation command-line arguments.
+   */
+  public static enum ExceptionAllocation {
+    /** Occurrence of exception should be considered an error */
+    ERROR,
+    /** Occurrence of exception should be considered invalid */
+    INVALID,
+    /** Occurrence of exception should be considered expected behavior */
+    REGRESSION
+  }
+  
+  /** The type of test to which a checked exception should be assigned. */
+  @Option("Type of test to which a checked exception is assigned")
+  public static ExceptionAllocation checked_exception = ExceptionAllocation.REGRESSION;
+  
+  /** The type of test to which an unchecked exception should be assigned. */
+  @Option("Type of test to which an unchecked exception is assigned")
+  public static ExceptionAllocation unchecked_exception = ExceptionAllocation.REGRESSION;
+  
+  /** The type of test to which a NullPointerException should be assigned */
```

Comment:
I would expand this Javadoc, which will go in the manual.  It's not for all NullPointerExceptions.
The `@Option` text can still be short.
However, for all of those, rather than "type of test ... should be assigned" which might be confusing, can you reword to be something more like how the behavior is viewed?